code/modules/power/supermatter/supermatter.dm

```diff
@@ -264,6 +264,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	internal_energy_factors = calculate_internal_energy()
 	zap_factors = calculate_zap_multiplier()
 	if(internal_energy && (last_power_zap + 4 SECONDS - (internal_energy * 0.001)) < world.time)
+		if(!has_been_powered)
+			log_activation(source = "the environment at [get_area_name(local_turf)]")
```

Comment:
won't this lead to us trying to get the key_name of this string?

Reply:
It will work text-wise but you just can't jump to it.

Reply:
hm